api/v1alpha1/virtualmachineimage_types.go

```diff
@@ -69,7 +70,7 @@ type VirtualMachineImageSpec struct {
 	ImageID string `json:"imageID"`
 
 	// ProviderRef is a reference to a content provider object that describes a provider.
-	ProviderRef ContentProviderReference `json:"providerRef"`
+	ProviderRef corev1.TypedLocalObjectReference `json:"providerRef"`
```

Comment:
Can you update the comment above to reflect the change here?

Reply:
Updated the comment to `ProviderRef is a reference to the source ContentLibraryItem/ClusterContentLibraryItem resource.`